submissions/Roka20012/pop-up-menu/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head> 
+    <meta charset="UTF-8">
```

Comment:
Don't forget about `meta` tag for mobile devices.

Reply:
Done)